civictechprojects/views.py

```diff
@@ -139,6 +147,8 @@ def get_group(request, group_id):
         return HttpResponse(status=404)
 
 
+@api_view('GET', 'POST')
```

Comment:
```suggestion
@api_view(['GET', 'POST'])
```
Page is failing to load with error message:
```
File "/app/civictechprojects/views.py", line 150, in <module>
@api_view('GET', 'POST')
TypeError: api_view() takes from 0 to 1 positional arguments but 2 were given
```